scripts/asm-installer/install_asm

```diff
@@ -2020,7 +2020,7 @@ EOF
 
   info "Installing ASM control plane..."
   # shellcheck disable=SC2086
-  retry 5 run "$(istioctl_path)" install --skip-confirmation $PARAMS
```

Comment:
Was this on purpose? If so can you note why?

Reply:
func istioctl_path will decide whether or not to use the _CI_ISTIOCTL_REL_PATH, we can leave this change into a separate PR, so we can discuss more there.

Reply:
I meant getting rid of the `--skip-confirmation` flag. I agree with the variable -> function change.